### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by leelapriyabhatta**
Added Spinner for export slides in activation page and changed hard coded texttospeech API string from website post service

### CODE REVIEW

**Review by ghchinoy — APPROVED** [review]
LGTM

**Review by ghchinoy — CHANGES_REQUESTED** [review]

File: frontend/src/app/content-review/content-review.component.scss [hunk 1]
```diff
@@ -422,5 +422,4 @@ st-editor {
     line-height: 20px;
     /* 142.857% */
     letter-spacing: 0.25px;
-    margin-top:20px
```
[the remark this answers is not in the document]
Comment (michaelwsherman) [reply frontend/src/app/content-review/content-review.component.scss:425]: Leela verbally confirmed that she will fix this.

[the remark this answers is not in the document]
Comment (leelapriyabhatta) [reply frontend/src/app/content-review/content-review.component.scss:425]: > Removal of this makes the button too close to the top separator Consider re-adding

I did add the css and pulled the changes from dev branch and removed unused css in this file

**Comment by awaemmanuel** [comment]
I don't still see a spinner @leelapriyabhatta. All I see is exported successfully at the bottom of page after waiting a while.

**Comment by awaemmanuel** [comment]
> I don't still see a spinner @leelapriyabhatta. All I see is exported successfully at the bottom of page after waiting a while.
> 
> image

I redeployed and see that the spinner is now working.

image

**Comment by awaemmanuel** [comment]
Tested the PR and spinner is present. I'll squash merge and close this PR. 

image

**Review by awaemmanuel — APPROVED** [review]
LGTM

**Review by ghchinoy — APPROVED** [review]
LGTM

**Pull request merged**